StreamChatClient_v3/Models/ChannelModel.swift

```diff
@@ -105,6 +108,31 @@ public struct ChannelModel<ExtraData: ExtraDataTypes> {
         self.isWatched = isWatched
         self.extraData = extraData
         self.invitedMembers = invitedMembers
+        self.messages = messages
+    }
+    
+    internal init(payload: ChannelPayload<ExtraData>) {
```

Comment:
You're probably aware of this but creating `ChannelModel` from `ChannelPayload` is not needed. All data should always go through the database. 
We need only `ChannelPayload` -> `ChannelDTO` and `ChannelDTO` -> `ChannelModel`